tensorflow/core/kernels/tensor_forest/tensor_forest.proto

```diff
@@ -0,0 +1,200 @@
+syntax = "proto3";
+
+package tensorflow.tensorforest;
+option cc_enable_arenas = true;
+option java_outer_classname = "TensorForestProtos";
+option java_multiple_files = true;
+option java_package = "org.tensorflow.framework";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/wrappers.proto";
+
+// Represents a single value of any type, e.g. 5 or "abc".
+message Value {
```

Comment:
Ok so historically this proto was chosen so any possible tree could be represented this way (eg universal format). When we were rewriting boosted trees, we decided to simplify and not to do this.
I strongly suggest to do the same. Just storing floats in the leaves should be enough
Please take a look at tensorflow/core/kernels/boosted_trees/boosted_trees.proto. can we reuse the same proto TreeEnsemble? For a forest, the weights would be 1/num trees and the trees between booosted trees and forest are the same (we will need to add a new split type, instead of bucketized split)